server/bin/pbench-backup-tarballs.py

```diff
@@ -248,17 +252,53 @@ def backup_to_s3(s3_obj, logger, controller_path, controller, tb, tar, resultnam
             _status = Status.FAIL
         return _status
 
-    tb_size = s3_obj.connector.getsize(tar)
-    if tb_size > (5 * (1024 ** 3)):
-        # FIXME: We will eventually implement multipart objects to take care
-        # of this but for now, we return a special failure so that the link is
-        # moved aside and we don't retry, but we save the links for eventual
-        # stashing into S3. Verification will complain about these.
-        logger.warning("Tar ball, {}, too large to upload", tar)
-        return Status.TOO_LARGE
-
     md5_base64_value = (base64.b64encode(
         bytes.fromhex(archive_md5_hex_value))).decode()
+    tb_size = s3_obj.connector.getsize(tar)
+    if tb_size > (5 * GB):
+        multi_upload_config = TransferConfig(
+                                            multipart_threshold=5*GB, multipart_chunksize=chunk_size)
+        # calculate multi etag value
+        local_multipart_etag = calculate_multipart_etag(tar, chunk_size)
+        try:
+            with open(tar, "rb") as f:
+                try:
+                    s3_obj.connector.upload_fileobj(f,
+                                                    Bucket=s3_obj.bucket_name,
+                                                    Key=s3_resultname,
+                                                    Config = multi_upload_config,
+                                                    ExtraArgs = {'Metadata': { 'MD5': local_multipart_etag}})
```

Comment:
I think there should be two pieces of metadata: the md5sum and the calculated etag.